docs/pages/setup/reference/networking.mdx

```diff
@@ -50,7 +50,8 @@ following use cases:
 
 Some networks funnel all connections through a proxy server where they can be
 audited and access control rules can be applied. For these scenarios, Teleport
-supports HTTP CONNECT tunneling.
+supports HTTP CONNECT tunneling for `tsh` and for `teleport` when running in
+[TLS routing mode](../../architecture/tls-routing.mdx).
```

Comment:
From this change, it sounds to me like the `tsh` and `teleport` binaries each support HTTP CONNECT in their own way, but is this more because `tsh` and `teleport` would be on either side of a client connection involving the Proxy Service?

I think we should make this paragraph more specific. Rather than saying "Teleport" supports HTTP Connect, we should talk about the specific Teleport services that are affected. This would be particularly useful for the `systemd` instructions below—I'm not sure if these apply to the `teleport` daemon that runs the Auth Service, Proxy Service, agents, etc.

And rather than include a brief link to explain TLS routing, I think we can incorporate the TLS routing setup instructions (i.e., editing `teleport.yaml`) into the instructions for enabling HTTP CONNECT tunneling via environment variables.